<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcTF</class>
 <widget class="QMainWindow" name="CalcTF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CalcTF</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="lbl_result">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_calc">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string>计算</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Start/Stop frequency sample interval</string>
        </property>
        <property name="text">
         <string>采集间隔(ms)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_freq_stop">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>5999999.988</string>
        </property>
        <property name="maxLength">
         <number>11</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Material density</string>
        </property>
        <property name="text">
         <string>材料密度</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="le_meas_itl">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>500</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_f_c">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Stop frequency</string>
        </property>
        <property name="text">
         <string>截至频率</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_matl_d">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>1.00</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_freq_start">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>6000000.000</string>
        </property>
        <property name="maxLength">
         <number>11</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="le_matl_z">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>1.00</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_f_q">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Start frequency</string>
        </property>
        <property name="text">
         <string>开始频率</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Z-Ratio</string>
        </property>
        <property name="text">
         <string>声阻抗</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
